### PR TITLE
Blacksmiths across the galaxy in shambles :: You can only forge with metals

### PR DESCRIPTION
**Pull request by Paxilmaniac**
## About The Pull Request

Adds a whitelist of materials that can be used in forging, which includes every metal and metal-like material we have. Why do this you ask? Because people forging with meat, wood, paper, whatever else you can think of that's **not** a metal seems rather quite silly to me.

## How This Contributes To The Skyrat Roleplay Experience

People won't be able to forge weapons and armor out of meat anymore.

## Proof of Testing

image

## Changelog

:cl:
fix: Forging can only be done with metals, and not bone/wood/meat/paper/diamonds/whatever else
/:cl:

### CODE REVIEW

**Review by tf-4 — APPROVED** [review]
No more diamond sword 😭

**Comment by Paxilmaniac** [comment]
> No more diamond sword 😭

Its joever

**Comment by Hatterhat** [comment]
its so joever

**Comment by kittysmooch** [comment]
speedclose. repository ban. dispatch hunter-killers droids

**Comment by Samarai** [comment]
I can understand removing silly materials like paper, cardboard, and meat. But this also lumps in bone, plasma, plasmaglass, glass, bluespace crystals, and other less silly materials for forging that look rather pretty when used.

**Comment by tf-4** [comment]
actually yeah add bone, that makes sense

**Comment by softcerv** [comment]
It would be amusing if later down the line, the ability to forge with things like paper and cardboard were reintroduced and ended up giving the items comedic effects. Using cloth to make a plush sword that does 0 damage would be funny.

**Comment by softcerv** [comment]
I think having sandstone included in this list would make sense.

**Comment by Paxilmaniac** [comment]
> I think having sandstone included in this list would make sense.

No, no I don't think so.

**Comment by HoodMilk** [comment]
the death of pizza sword

File: modular_skyrat/modules/reagent_forging/code/forge_items.dm [hunk 1]
```diff
@@ -1,3 +1,26 @@
+GLOBAL_LIST_INIT(allowed_forging_materials, list(
```
Comment (ghost) [inline modular_skyrat/modules/reagent_forging/code/forge_items.dm:1]: should this be alphabetized or otherwise sorted?

Comment (Paxilmaniac) [reply modular_skyrat/modules/reagent_forging/code/forge_items.dm:1]: It is sorted just in "oh right I need to add that material too" order

**Pull request merged**